### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by Yi0724**
45209484
Yi Yang
Assignment 3 uploaded.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be more specific
- No dedicated test set, only train and val
- Results and code appear okay for Easy Difficulty

Other Feedback
-Good overall. would put main logic in `if __name__ == '__main__'` for convention sake

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

You pull request should have a description of what it is and why it should be merged 

Tensorflow has functions for image resizing and more so there’s no need to also use PIL in your project. 

Comments such as 
    # fit
    fit(model,x_train,y_train,10,4)
Add nothing to the readability of the code and should be avoided. But overall comments are good. 

Code in files such as main.py should be in a main function.

File: recognition/45209484-yi yang/functions.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/45209484-yi yang/functions.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
